apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -302,15 +329,15 @@ def compute_improved_policy(
     model_n_logits: int,
     N_bar: int,
 ):
-    """Compute the improved policy given the state_dict, the list of states. The
-    improved policy is computed as (N_s_a / N_s_a.sum())ˆ(1/tau) where tau is
-    (log(N_s_a.sum()) / log(N_bar))
+    """Compute the improved policy given the state_dict, the list of states.
+    The improved policy is computed as (N_s_a / N_s_a.sum())ˆ(1/tau) where tau
+    is (log(N_s_a.sum()) / log(N_bar))
     """
     policies = torch.zeros(len(states), model_n_steps, model_n_logits)
     N_bar = torch.tensor(N_bar)
     for idx, state in enumerate(states):
-        N_s_a = state_dict[state][3]
-        actions = state_dict[state][5]
+        N_s_a = state_dict[state][2]
+        actions = state_dict[state][4]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)